README.md

```diff
@@ -1,2 +1,25 @@
-# WorklistTool-GSoC-2018
-Tool is being developed for Wikimedia under GSoC 2018
+# Worklist Tool
+The Worklist Tool is a tool for creating lists of Wikimedia pages which users can then assign themselves to for claiming and tracking work during editing events and programs.
+
+The tool was first developed with Wikimedia during Google Summer of Code 2018.
+
+## Usage
+The Worklist Tool is currently hosting on Toolforge at https://tools.wmflabs.org/worklist-tool/.
```

Comment:
Just "The Worklist Tool is hosted on Toolforge" ... ?

(Also we should consider a catchier name, but don't have to do it now).

Reply:
Yep, that was some poor wording - fixed.

Reply:
Yep, that was some poor wording - fixed.